.travis.yml

```diff
@@ -23,60 +25,6 @@ stages:
 matrix:
   fast_finish: true
   include:
```

Comment:
As it is, acceptance tests will be not executed on travis. 
The entire bloc code was removed. For that, we need to update .sync file to include it there and then to run again a pdk update to include the chnages also in travis

Cheers!